Classes/Neos/Neos/Ui/FlowQueryOperations/NeosUiDefaultNodesOperation.php

```diff
@@ -17,7 +17,7 @@
 use Neos\ContentRepository\Domain\Model\NodeInterface;
 
 /**
- * Renders nodes that are initially required by the user interface
+ * Fetches all nodes needed for the given state of the UI
```

Comment:
isn't this function actually *slower* than the one before? :)

Reply:
For default state, with no uncollapsed nodes, it's equivalent. It includes nodes on the rootline + all uncollapsed nodes so extra requests would not be needed when opening on a deep nested node, deeper then loadingDepth, or when reloading the tree.